tests/test_table_row_classification.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+feature_table_dict = {
+    "area": np.array([10, 10, 30, 30, 10]),
+    "surface_area": np.array([5, 3, 80, 80, 9])
+}
+feature_table_data_frame = pd.DataFrame(feature_table_dict)
+ground_truth = np.array([1, 1, 2, 2, 1])
+
+
+@pytest.mark.parametrize("feature_table", [feature_table_dict, feature_table_data_frame])
```

Comment:
@haesleinhuepf , here I used the `parametrize` fixture in pytest to use the same test, but pass in different feature table types. https://docs.pytest.org/en/6.2.x/parametrize.html